framework/PageInputs/PageSingleSelect.tsx

```diff
@@ -162,42 +162,49 @@ export function PageSingleSelect<
 
   const selectedLabel = useMemo(() => {
     let selectedLabel: ReactNode = selectedOption?.label;
-    if (!selectedLabel && value !== undefined && value !== '') {
+    if (!selectedLabel && value !== undefined && value !== '' && value !== null) {
```

Comment:
what about simplification:
!selectedLabel && value

Reply:
If value was 0 that would fail.

Reply:
Oh, so it is not only string, makes sense.